README.md

```diff
@@ -28,6 +28,12 @@ var obj = {
 fixturify.writeSync('testdir', obj) // write it to disk
 
 fixturify.readSync('testdir') // => deep-equals obj
+
+fs.removeSync('testDir', {
+    'subdir': { 'bar.txt' }
+  }) // remove subdir/bar.txt
```

Comment:
I'm a bit confused about how exactly `removeSync` is supposed to work.

* I assume you meant `fixturify.removeSync`?
* And `{ 'bar.txt' }` doesn't look right...